templates/zerver/api/outgoing-webhooks.md

```diff
@@ -77,25 +94,65 @@ would be able to post data to URLs which support Slack's outgoing
 webhooks.  Here's how we fill in the fields that a Slack format
 webhook expects:
 
-```
-token:          string of alphanumeric characters (similar to above)
-team_id:        string id of the Zulip organization
-team_domain:    domain of the Zulip organization
-channel_id:     stream id
-channel_name:   stream name
-timestamp:      timestamp when message was sent
-user_id:        id of user who sent the message
-user_name:      full name of sender
-text:           content of message in a more readable format
-trigger_word:   trigger method
-service_id:     id of bot user
-```
```

Comment:
We should have an example down here, too, right?

Reply:
No, because this data is passed as tuples as part of the URL and not as JSON, so I think not!

Reply:
I'm not 100% sure of what do you mean with tuples in the URL, but if it's tuple-like strings encoded as a querystring it may be worth showing a key-value pair, even if it's only a small code block showing how it looks. Some other people may find it a bit confusing as well.